src/commands/app/logs.ts

```diff
@@ -22,21 +22,47 @@ export default class AppLogs extends Command {
       char: 's',
       description: 'show logs since timestamp',
     }),
+    timestamps: Flags.boolean({
+      char: 't',
+      description: 'Show timestamps',
```

Comment:
To be consistent with the rest of the project, let's use lowercase descriptions:
```suggestion
      description: 'show timestamps',
```